src/js/badge.js

```diff
@@ -0,0 +1,48 @@
+import {isBadgeNonexisted} from './common/domUtil';
+import $ from 'jquery';
+
+const selectors = [
+  'h1.public > strong > a',                             //Specific repository
+  '.js-repo-filter .d-inline-block h3 a:first-child',   //Repositories & Stars
+  '.org-repos .d-inline-block a',                       //Organization
+  '.codesearch-results .d-inline-block a',              //Search
+  '.explore-pjax-container .explore-content a'          //Trending
+];
+const repoListSelectors = '.js-pinned-repos-reorder-container p.mb-0'; //Overview
+
+const showBadge = () => {
+  selectors.forEach((sel) => {
+    $(sel).each((i, element) => {
+      insertStatusBadge(element, element.pathname);
+    });
```

Comment:
That's a very interesting question. Actually, $(sel) is a jquery object, so we cannot use forEach for it. At first I use the code as below, 
```
$(sel).each(() => {
      insertStatusBadge(this, this.pathname);
});
```
but `this` would be transformed into `undefine` by babel in this situation.